content/home.js

```diff
@@ -11,8 +11,69 @@ module.exports = {
+  },
+  articleThree: {
+    title: 'Key insight 3',
+    text:
+      'Nullam eget felis eget nunc lobortis. Faucibus ornare suspendisse sed nisi. Sagittis eu volutpat odio facilisis mauris sit amet massa. Erat velit scelerisque in dictum non consectetur a erat. Amet nulla facilisi morbi tempus iaculis urna. Egestas purus viverra accumsan in nisl. Feugiat in ante metus dictum at tempor commodo.\n' +
+      'Convallis tellus id interdum velit laoreet. Proin sagittis nisl rhoncus mattis rhoncus urna neque viverra. Viverra aliquet eget sit amet tellus cras adipiscing enim eu. Ut faucibus pulvinar elementum integer enim neque volutpat ac tincidunt.',
+    articleImage: '/images/articles/article3.jpg',
+    conclusion:
+      'Dui faucibus in ornare quam. In iaculis nunc sed augue lacus viverra vitae congue. Vitae tempus quam pellentesque nec nam aliquam sem et. Ut morbi tincidunt augue interdum. Sem fringilla ut morbi tincidunt augue.',
+  },
+  articleFour: {
+    title: 'Key insight 4',
+    text: 'Dui faucibus in ornare quam. In iaculis nunc sed augue lacus viverra vitae congue. Vitae tempus quam pellentesque nec nam aliquam sem et. Ut morbi tincidunt augue interdum. Sem fringilla ut morbi tincidunt augue. Morbi enim nunc faucibus a pellentesque sit amet porttitor eget. In est ante in nibh mauris. Nam aliquam sem et tortor consequat id porta nibh. Diam quis enim lobortis scelerisque fermentum dui faucibus. Non curabitur',
+    conclusion:
+      'Learn more in the full report, lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. ',
+  },
+  quoteOne: {
+    text: 'Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores.',
+    authorTitle: `Author name <br/> Designation, Company name`,
+    quoteImg: './images/articles/quotes.png',
+  },
   common: {
     arrowDown: '/images/common/arrowDown.png',
+    arrowWhite: '/images/common/arrowWhite.png',
+    prevArrow: '/images/navigation/prev.png',
+    nextArrow: '/images/navigation/next.png',
+  },
+  navigationPrev: {
+    navigationImage: '/images/navigation/navigation1.jpg',
+    title: 'Gold mid-year outlook 2021',
+    text: 'Lorem ipsum dolor sit amet, consec tetur adipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et.',
+    direction: 'Previous',
+    prevMobile: '/images/common/prevMobile.png',
+    mobileText: 'Prev',
+  },
+  navigationNext: {
+    navigationImage: '/images/navigation/navigation2.jpg',
+    title: 'Gold Market Commentary',
+    text: 'Lorem ipsum dolor sit amet, consec tetur adipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et.',
+    direction: 'Next',
+    nextMobile: '/images/common/nextMobile.png',
+    mobileText: 'Next',
```

Comment:
the text for both mobile and desktop direction buttons (next, prev) are universal, they can be moved into the component itself